Sources/BenchmarkSupport/BenchmarkRunner.swift

```diff
@@ -47,10 +47,10 @@ public struct BenchmarkRunner: AsyncParsableCommand, BenchmarkRunnerReadWrite {
     var debug = false
 
     func shouldRunBenchmark(_ name: String) throws -> Bool {
```

Comment:
These changes flowed into place when I ran `swiftformat .` using the configuration within the repo. I presumed that was desired when submitting code changes. If not, I can back these out.